shell_integration/dolphin/CMakeLists.txt

```diff
@@ -43,6 +40,8 @@ add_library(${OWNCLOUDDOLPHINHELPER} SHARED ownclouddolphinpluginhelper.cpp)
 target_link_libraries(${OWNCLOUDDOLPHINHELPER} Qt5::Network Qt5::Gui)
 generate_export_header(${OWNCLOUDDOLPHINHELPER} BASE_NAME ownclouddolphinpluginhelper)
 install(TARGETS ${OWNCLOUDDOLPHINHELPER} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+target_include_directories(${OWNCLOUDDOLPHINHELPER} PUBLIC ${CMAKE_CURRENT_BINARY_DIR} ${PROJECT_BINARY_DIR} ${PROJECT_BINARY_DIR}/src)
```

Comment:
https://cmake.org/cmake/help/latest/variable/CMAKE_INCLUDE_CURRENT_DIR.html
Will be needed in the dolphin CMakeLists.txt too as it has its own project() scope.
But should be covered by https://invent.kde.org/frameworks/extra-cmake-modules/-/blob/master/kde-modules/KDECMakeSettings.cmake#L194

Reply:
Let me check whether it works without this dir.

I think in Qt projects, `CMAKE_INCLUDE_CURRENT_DIR` should be set to `On` anyway.

Reply:
It is through KDE settings